packages/editor/src/components/editor-help/help-detail-navigation-screen.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * External dependencies
+ */
+import { View, Text } from 'react-native';
+
+/**
+ * WordPress dependencies
+ */
+import { BottomSheet } from '@wordpress/components';
+import { useNavigation } from '@react-navigation/native';
+
+/**
+ * Internal dependencies
+ */
+import styles from './style.scss';
+import { __ } from '@wordpress/i18n';
+
+const HelpDetailNavigationScreen = ( { name } ) => {
+	const navigation = useNavigation();
+
+	const goBack = () => {
+		navigation.goBack();
+	};
+
+	return (
+		<BottomSheet.NavigationScreen name={ __( 'What is a block?' ) }>
```

Comment:
I would think this `name` property needs to be dynamic to match the name of the screen itself, i.e. set to `name`. My understanding is that `name` is the string label used to select _which_ screen to navigate to when calling `navigation.navigate(<screen-name>)`. I am a bit puzzled as to how this is working currently, but it does appear to work as-is. WDYT? 

Note: This point would like become moot if we decide to use `BottomScreen.SubSheet`.

Reply:
I believe it worked because each sheet would have a different name (albeit localized). Doesn't matter at this point though since I did end up using `BottomSheet.SubSheet` thanks to your suggestion 👍